beaker-docker.gemspec

```
@@ -27,6 +27,10 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rspec-its', '~> 1.3'
   s.add_development_dependency 'fakefs', '~> 1.3'
   s.add_development_dependency 'rake', '~> 13.0'
+  s.add_development_dependency 'rubocop', '~> 1.12'
```

I'd recommend to pin to a .y version because Rubocop doesn't really follow semver.
```suggestion
  s.add_development_dependency 'rubocop', '~> 1.12.0'
```

That said, if you drop Ruby 2.4 support you can use 1.28.x (see https://docs.rubocop.org/rubocop/compatibility.html):
```suggestion
  s.add_development_dependency 'rubocop', '~> 1.28.0'
```

---

@bastelfreak said above we should target 2.4 because the gem supports it 🤔